client/src/components/FormikDatePicker/FormikDatePicker.tsx

```diff
@@ -0,0 +1,122 @@
+import DatePicker from '@mui/lab/DatePicker';
+import { SelectChangeEvent, Select, Box, InputLabel, InputBase, MenuItem } from '@mui/material';
+import { styled } from '@mui/material/styles';
+import { format, hoursToMilliseconds } from 'date-fns';
+import { useState } from 'react';
+
+const StyledInput = styled(InputBase)(({ theme }) => ({
+  'label + &': {
+    marginBottom: theme.spacing(3),
+  },
+  '& .MuiInputBase-input': {
+    fontSize: 16,
+    width: '100%',
+    padding: '15px',
+  },
+}));
+
+interface Props {
+  dateField: string;
+  date: Date;
+  setFieldValue: (field: string, value: any) => void;
```

Comment:
Won't the `value` be a Date type?

Reply:
Apparently datepicker passes Date | null, so I changed it to that.